pkg/apis/core/v1/conversion.go

```diff
@@ -60,6 +60,7 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 			switch label {
 			case "metadata.name",
 				"metadata.namespace",
+				"metadata.ownerReferences.uid",
```

Comment:
Did you mean metadata.uid?

Reply:
Sorry, I reread the PR description and you did mean this. The actual path would have to be `metadata.ownerReferences[*].uid` or something like that. This would be a very new sort of filter, I don't see how it can be made to work with the filtering stack.